src/monster.cpp

```diff
@@ -1469,12 +1469,12 @@ int monster::impact( const int force, const tripoint &p )
     const float mod = fall_damage_mod();
     int total_dealt = 0;
     if( g->m.has_flag( TFLAG_SHARP, p ) ) {
-        const int cut_damage = 10 * mod - get_armor_cut( bp_torso );
+        const int cut_damage = std::max( 0.0f, 10 * mod - get_armor_cut( bp_torso ) );
```

Comment:
Sometimes it complains if the choice is between `std::max<float>` and `std::max<double>` (or `int` or whatever).